### ISSUE TITLE
generate graphs of metrics for all models (f1-score; accuracy; recall; precision). Look at sklearn.metrics (esp. classification_report)

### ISSUE THREAD

**Issue by paulknewton** [paulknewton/twitter-ml#27]

**Comment by paulknewton** [comment]
Not relevant. Metrics are simply numbers so cannot be plotted. Already displayed with the --report option.
